app/models/apple/config.rb

```diff
@@ -103,27 +103,18 @@ def self.setup_delegated_delivery(podcast, key: nil, apple_config: nil, apple_sh
       mark_as_delivered!(pub)
     end
 
-    def one_config_per_podcast
-      return unless public_feed.present? && private_feed.present?
-      return unless podcast.present?
-
-      if podcast.feeds.map(&:apple_configs).flatten.compact.present?
-        errors.add(:podcast, "can only have one apple config")
-      end
-    end
-
     def feed_podcasts_match
       return unless public_feed.present? && private_feed.present?
 
-      if public_feed.podcast != private_feed.podcast
+      if (public_feed.podcast_id != podcast_id) || (private_feed.podcast_id != podcast_id)
```

Comment:
Good to see that we're validating, the podcasts / feeds line up hierarchically under the podcast_id